Sources/Exporters/DatadogExporter/Persistence/FilesOrchestrator.swift

```diff
@@ -183,3 +183,24 @@ internal func fileCreationDateFrom(fileName: String) -> Date {
     let millisecondsSinceReferenceDate = TimeInterval(UInt64(fileName) ?? 0) / 1_000
     return Date(timeIntervalSinceReferenceDate: TimeInterval(millisecondsSinceReferenceDate))
 }
+
+private enum FixedWidthIntegerError<T: BinaryFloatingPoint>: Error {
```

Comment:
cannot find the use of this new enum. Is it part of this PR?

Reply:
yes is only used privately in this Datadog exporter module, and is used in line 202.
This was added here after removing SwiftExtension file because some of those hekpers were moved to Api